drawbot/utils/pronote.py

```diff
@@ -22,7 +22,7 @@ def fetch_homeworks(pronote_client: pronotepy.Client) -> Optional[Generator]:
 
 
 def fetch_grades(pronote_client: pronotepy.Client) -> Optional[Generator]:
-    fetched_grades = pronote_client.periods[0].grades
+    fetched_grades = pronote_client.current_period.grades
```

Comment:
Nice! It works fine (wasn't aware of that, and I think I just did `[0]` for testing purpose and forget to search the real production method). Anyway, thanks for your help :D